samples/hono/src/handlers/createPets.ts

```diff
@@ -0,0 +1,12 @@
+import { createFactory } from 'hono/factory';
```

Comment:
The handlers is  sccafold and i have to implement it myself after generation right?
In that case, I understand that regenerating will roll back the implementation, is this my correct understanding?

Reply:
Indeed you will need to implement it your self and I added some verification to check if the handler name is in the file or not like that I don't scrap your implementation if it's already in the handler file

Reply:
> I don't scrap your implementation if it's already in the handler file

that's great 👍  I had missed that implementation.